src/library/modules/TsunDBSubmission.js

```diff
@@ -1320,6 +1321,17 @@
 						}
 
 						misc = buildSortieSpecialInfo(fleet, cutinType[1]);
+						
+						// Sending the attack round to check if the trigger rates from the first and second round are different
+						misc.attackRound = num;
+						
+						// Sending helper HP info to check if chuuha affects touch trigger rate
+						if (num === 0) for (let idxk = 0; idxk < shipIndexListSpecial[cutinType[1]].length; idxk++)
```

Comment:
`shipIndexListSpecial[cutinType[1]]` are outside of if statement above, so undefined.length may happen?

Reply:
Thanks, I have fixed that.